src/main/java/org/prebid/server/bidder/telaria/TelariaBidder.java

```diff
@@ -145,20 +146,14 @@ private ExtImpTelaria parseImpExt(Imp imp) {
 
     private Site modifySite(String seatCode, BidRequest bidRequest) {
         final Site site = bidRequest.getSite();
-        if (site.getPublisher() != null) {
-            final Publisher modifiedPublisher = site.getPublisher().toBuilder().id(seatCode).build();
-            return site.toBuilder().publisher(modifiedPublisher).build();
-        }
-        return site;
+        final Publisher modifiedPublisher = site.getPublisher().toBuilder().id(seatCode).build();
+        return site.toBuilder().publisher(modifiedPublisher).build();
     }
 
     private App modifyApp(String seatCode, BidRequest bidRequest) {
         final App app = bidRequest.getApp();
-        if (app.getPublisher() != null) {
-            final Publisher modifiedPublisher = app.getPublisher().toBuilder().id(seatCode).build();
-            return app.toBuilder().publisher(modifiedPublisher).build();
-        }
-        return app;
+        final Publisher modifiedPublisher = app.getPublisher().toBuilder().id(seatCode).build();
+        return app.toBuilder().publisher(modifiedPublisher).build();
```

Comment:
NPE
 site can be null.
app can be null.
Publisher can be null.

Please, just create separate method that can create publisher, and checks for NULL. 
Call this method from modifySite and modifyApp method that get Site and App object accordingly. 
ALSO performing checks for null.